flow-report/assets/styles.css

```diff
@@ -26,6 +26,10 @@
   font-size: var(--app-font-size);
 }
 
+.Content {
+  overflow-y: scroll;
```

Comment:
rationale for scroll vs. auto here?

Reply:
`scroll` will force scrollbars to always appear (but if you're on macOS that doesn't always mean much ;)) whereas `auto` will only add scrollbars if the content requires scrolling.

The tradeoff here is that auto looks "cleaner" when there's no scrolling but there might be layout shift due to the track when content lengthens.

Reply:
`auto` sounds better then, since the content should only change size because of user interaction.